### PR TITLE
feat(cli): first few cli commands

### PR DESCRIPTION
**Pull request by dgozman**
This sets up cli capabilities and brings in a few commands from playwright-cli.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,10 +18,6 @@
...
 setUnderTest(); // Note: we must call setUnderTest before initializing.
-
-const { Playwright } = require('./lib/server/playwright');
-const { setupInProcess } = require('./lib/inprocess');
-const path = require('path');
-
-const playwright = new Playwright(__dirname, require(path.join(__dirname, 'browsers.json'))['browsers']);
-module.exports = setupInProcess(playwright);
+module.exports = require('./lib/inprocess');
+if (require.main === module)
```
Comment (JoelEinbinder) [inline index.js:22]: this is a strange python pattern. The standard way ive seen is to set the "bin" field in `package.json` with a separate file. See https://github.com/microsoft/folio/blob/master/package.json#L4

File: packages/installation-tests/installation-tests.sh [hunk 2]
```diff
@@ -40,18 +40,20 @@ function copy_test_scripts {
...
-  test_playwright_global_installation_subsequent_installs
-  test_playwright_should_work
-  test_playwright_chromium_should_work
-  test_playwright_webkit_should_work
-  test_playwright_firefox_should_work
-  test_playwright_global_installation
-  test_playwright_global_installation_cross_package
-  test_playwright_electron_should_work
-  test_electron_types
+  # test_screencast
```
Comment (JoelEinbinder) [inline packages/installation-tests/installation-tests.sh:43]: just for the draft?

File: packages/installation-tests/sanity.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ const installer = require(requireName + '/lib/install/installer');
       await browser.close();
     } catch (e) {
       console.error(`Should be able to launch ${browserType} from ${requireName}`);
-      console.error(err);
+      console.error(e);
```
Comment (JoelEinbinder) [inline packages/installation-tests/sanity.js:45]: :+1:

File: packages/installation-tests/installation-tests.sh (hunk 2 above)
Comment (dgozman) [reply packages/installation-tests/installation-tests.sh:43]: Thanks for catching this!

File: index.js (hunk 1 above)
Comment (dgozman) [reply index.js:22]: Done.

**Comment by dgozman** [comment]
@JoelEinbinder Please take another look.

File: package.json [hunk 4]
```diff
@@ -38,7 +38,11 @@
     "name": "Microsoft Corporation"
   },
   "license": "Apache-2.0",
+  "bin": {
+    "playwright": "./lib/cli/cli.js"
```
Comment (yury-s) [inline package.json:42]: Do we want to merge playwright-cli npm into playwright? I thought we wanted to keep them separate.

**Review by yury-s — APPROVED** [review]

File: package.json (hunk 4 above)
Comment (dgozman) [reply package.json:42]: I think that's the point of merging!

**Pull request merged**